numpy/random/mtrand.pyx

```diff
@@ -25,15 +25,15 @@ from ._common cimport (POISSON_LAM_MAX, CONS_POSITIVE, CONS_NONE,
             check_array_constraint, check_constraint, disc, discrete_broadcast_iii,
         )
 
-cdef extern from "numpy/random/distributions.h":
+cdef extern from "include/distributions-long.h":
     struct s_binomial_t:
         int has_binomial
         double psave
-        int64_t nsave
+        long nsave
         double r
         double q
         double fm
-        int64_t m
+        long m
         double p1
         double xm
         double xl
```

Comment:
cython doesn't seem to care what `int` is used in the struct, it is later filled out by the compiler. But cleaning this up anyway for consistency.